e2e_test/udf/sql_udf.slt

```diff
@@ -342,6 +348,116 @@ create function add_error(INT, INT) returns int language sql as $$select $1 + $2
 # statement error function fib calling stack depth limit exceeded
 # create materialized view bar_mv as select fib(c1) from t1;
 
+# 2. Select from table inside SQL UDF (potential concerns: naming conflict)
+
+# statment ok
+# create funciton from_table_single_row() returns int language sql as 'select c1 from t1';
+
+# Do NOT need explicit `select ... from ...`
+# query I
+# select from_table_single_row();
+# ----
+# 1
+
+# statement ok
+# create function from_table_single_row_1() returns int language sql as 'select c1 from t1 order by c1 desc';
+
+# Need explict `select ... from ...`
+# query I
+# select * from from_table_single_row_1();
+# ----
+# 5
+
+# Should add parser support
+# statement ok
+# create function from_table_multiple_rows() returns setof int language sql as 'select c1 from t1';
```

Comment:
```suggestion
# create function from_table_multiple_rows() returns table (c1 int) language sql as 'select c1 from t1';
```

Reply:
Just curious, what's the concern to return table instead of a set of integers here? 🤔

Reply:
Sorry. I didn't know `returns setof` is a valid syntax. 🥵

Reply:
Presumably we could use this two syntax interchangeably, let me do some further investigations to determine which goes first. 😄

Reply:
I guess `returns setof int` can be desugared to `returns table (function_name int)`.